docs/configuration/providers.md

```diff
@@ -0,0 +1,38 @@
+## Providers
+
+### GitHub Enterprise
+
```

Comment:
Can you add here the gifs from this page that might be relevant for me to complete the setup?
https://support.codacy.com/hc/en-us/articles/207280489-Configuring-GitHub-Enterprise-with-Codacy-Self-hosted